### ISSUE TITLE
Tabs are not keyboard accessible, nor follow accessibility standards

### ISSUE THREAD

**Issue by jasonday** [requarks/wiki#4461]
### Actual behavior
Unable to access or activate tabs via keyboard. 

### Expected behavior
The ability to access and activate tabs via keyboard.

The issue is that Wiki.js is using `<li>` for interactive elements, which is semantically incorrect. 
```
<ul class="tabset-tabs">
    <li class="is-active">Usage</li>
    <li class="">Shortcuts</li>
    <li class="">Examples</li>
    <li class="">Stylings</li>
</ul>
```
And the component does not include the necessary attributes to make the component accessible to screen readers and other assistive technologies beyond the inability to keyboard access. 

See: https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

Additionally, there are other high user impacting accessibility issues throughout the site - are there plans to improve wiki.js's accessibility?

**Comment by NGPixel** [comment]
Fixed by 63ecb32494472480ebc0bce9282f071007e11692 and will be part of the next hotfix release.

Added all attributes and controls listed in the link your provided.

Tabs are now properly focusable and can be controlled via <kbd>Left</kbd>/<kbd>Right</kbd> arrows, <kbd>Enter</kbd>/<kbd>Space</kbd>, <kbd>Home</kbd> and <kbd>End</kbd> keys.

Accessibility should improve in v3, but feel free to report any other major accessibility issues for v2 in the meantime.
